utils/build/docker/nodejs/express4/app.js

```diff
@@ -134,13 +135,64 @@ app.get("/users", (req, res) => {
   }
 });
 
+app.get("/dsm", (req, res) => {
+  const kafka = new Kafka({
+    clientId: 'my-app',
+    brokers: ['kafka:9092'],
+    retry: {
+      initialRetryTime: 100, // Time to wait in milliseconds before the first retry
+      retries: 20, // Number of retries before giving up
+    },
+  })
```

Comment:
is this supposed to be executed for every request ? shouldn't it be initialized at the top level ? I'm not sure tbh i never ~read~ used kafka